Makefile

```diff
@@ -249,6 +255,9 @@ run-compose: images/docker-compose.yml images/version.yml
 	docker-compose -f $< run storage-init sh -c 'rm -rf /run/* /config/* ; cp -Lr /conf/* /config/ ; echo IMGA > /run/eve.id'
 	docker-compose -f $< up
 
+run-proxy:
+	ssh $(SSH_PROXY) -N -i $(CONF_DIR)/ssh.keys -p $(SSH_PORT) -o StrictHostKeyChecking=no -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null root@localhost &
```

Comment:
I didn't actually understand why we need the proxy? If adam (CLI) can talk to 6000 localhost (proxy locally), and the proxy can talk to 2222 on the device (via qemu's forwarding), can we not just enable 6000 forwarding via qemu when running with adam?

Reply:
As it is, the proxy tends to crash when it has no connection for some period of time.

Reply:
Just update [this line](https://github.com/lf-edge/eve/blob/master/Makefile#L138) from:

```
        -netdev user,id=eth0,net=$(QEMU_OPTS_NET1),dhcpstart=$(QEMU_OPTS_NET1_FIRST_IP),hostfwd=tcp::$(SSH_PORT)-:22 -device virtio-net-pci,netdev=eth0 \
```

to

```
        -netdev user,id=eth0,net=$(QEMU_OPTS_NET1),dhcpstart=$(QEMU_OPTS_NET1_FIRST_IP),hostfwd=tcp::$(SSH_PORT)-:22,hostfwd=tcp::6000-:6000 -device virtio-net-pci,netdev=eth0 \
```

Probably should put `6000` in a variable, and I might have typoed on the syntax, but it gets the idea.

Reply:
Well, the problem here is that port 6000 is blocked by EVE when it comes to traffic coming from "outside".

EVE only ever makes 22 open (and even that has to be driven by a local config item that our local Adam instance has to give to EVE). Once 22 is open -- that becomes our only chance to proxy any other connection to appear local to EVE.

Makes sense @deitch ?

Reply:
Running embedded adam is for local building and testing anyways, so can we not have the build with `ADAM=y` open that port?